records_mover/airflow/hooks/sqlalchemy_db_hook.py

```diff
@@ -4,15 +4,15 @@
 
 
 class SqlAlchemyDbHook(BaseHook):
-    def __init__(self, db_conn_id):
+    def __init__(self, db_conn_id: str) -> None:
         self.db_conn_id = db_conn_id
 
-    def get_conn(self):
+    def get_conn(self) -> sa.engine.Engine:
         conn = BaseHook.get_connection(self.db_conn_id)
         db_url = create_sqlalchemy_url(
             {
                 'host': conn.host,
-                'port': str(conn.port),
+                'port': conn.port,
```

Comment:
This wasn't necessary - we take port as an int everywhere else and pass it to `create_sqlalchemy_url()` which accepts an int - you can run `db-facts json redshift` to see that port is kept as an int.  No idea why someone added the cast originally.